docs/api.md

```diff
@@ -336,7 +336,7 @@ You can expect the following responses from this API:
 
 -   `412` - The client requested a precondition that cannot be satisfied.
 
--   `429` - The client is rate limited. Check the `X-Retry-After` header to
```

Comment:
There is no `x-` for retry-after
https://github.com/sethvargo/go-limiter/blob/86d882305d36ae8c59138bc9a93b3599f0b70936/httplimit/middleware.go#L28

Reply:
Why would I remember my own code? 😝